Library/ViewModels/ThanksViewModel.swift

```diff
@@ -198,47 +198,62 @@ public final class ThanksViewModel: ThanksViewModelType, ThanksViewModelInputs,
       checkoutData: $0.checkoutData
     ) }
 
-    // FB CAPI
-
+    // Facebook CAPI + Google Analytics
     _ = Signal.combineLatest(
       project,
       self.configureWithDataProperty.signal.skipNil()
     )
     .observeValues { project, configData in
-      var checkoutUSDAmount = ""
+      var pledgeAmount: Double?
+      var shipping: Double?
+      var transactionId: String?
 
       if let checkoutDataValues = configData.checkoutData {
-        checkoutUSDAmount = "\(checkoutDataValues.revenueInUsd)" + "\(checkoutDataValues.addOnsMinimumUsd)"
-        checkoutUSDAmount += String(describing: checkoutDataValues.shippingAmountUsd)
-        checkoutUSDAmount += String(describing: checkoutDataValues.bonusAmountInUsd)
+        transactionId = checkoutDataValues.checkoutId
+        shipping = checkoutDataValues.shippingAmountUsd
+        pledgeAmount = Double(truncating: checkoutDataValues.revenueInUsd as NSNumber)
+          + Double(truncating: (checkoutDataValues.bonusAmountInUsd ?? 0) as NSNumber)
+          + checkoutDataValues.addOnsMinimumUsd
       }
```

Comment:
I didn't verify this - but in case we lose some precision here - this might be more accurate because it doesn't "truncate": 

```
      if let checkoutDataValues = configData.checkoutData {
        transactionId = checkoutDataValues.checkoutId
        shipping = checkoutDataValues.shippingAmountUsd
        pledgeAmount = NSDecimalNumber(decimal: checkoutDataValues.revenueInUsd).doubleValue
          + NSDecimalNumber(decimal: checkoutDataValues.bonusAmountInUsd ?? 0).doubleValue
          + checkoutDataValues.addOnsMinimumUsd
      }
```